src/Raven.Server/Utils/BackupUtils.cs

```diff
@@ -35,6 +35,54 @@ namespace Raven.Server.Utils;
 
 internal static class BackupUtils
 {
+    internal static BackupTask GetBackupTask(DocumentDatabase database, BackupParameters backupParameters, BackupConfiguration configuration, Logger logger, PeriodicBackupRunner.TestingStuff forTestingPurposes = null)
+    {
+        return IsDirectUploadSupported(out var directUploadDestination) 
```

Comment:
I do not like that it silently switches to regular one, shouldn't we validate that when backup is being created and only do a sanity check here?